### PR TITLE
CB-11072: Avoid the need to stop ranger and HMS services before performing restore.

### PR DESCRIPTION
**Pull request by kkalvagadda1**
While performing backup and restore we need to make sure that there are no updates to metadata to make sure that we have a consistent backup. Currently, it is documented but there is no way to enforce it.

This approach will make sure that HMS and Ranger's data is updated while backup-restore operations are performed on data lake.

This change basically does below

1. Limit the number of connections to the database to 0.
2. Close all the active connections to the database.
3. perform the restore operation.
4. Limit the number of connections to the database to -1 (unlimited)

### CODE REVIEW

File: orchestrator-salt/src/main/resources/salt/salt/postgresql/disaster_recovery/scripts/restore_db.sh [hunk 1]
```diff
@@ -112,7 +129,7 @@ run_restore() {
   then
     STRIPPED_S3_URL=$(echo "$BACKUP_LOCATION" | sed -e 's/[a-fA-F0-9]\{8\}-[a-fA-F0-9]\{4\}-[a-fA-F0-9]\{4\}-[a-fA-F0-9]\{4\}-[a-fA-F0-9]\{12\}_database_backup\/*\*\?$//g')
     doLog "Begining to perform s3guard import for ${STRIPPED_S3_URL}"
-    hadoop s3guard import "$STRIPPED_S3_URL" > >(tee -a $LOGFILE) 2> >(tee -a $LOGFILE >&2) || doLog "ERROR Unable to import S3Guard metadata."
+ #   hadoop s3guard import "$STRIPPED_S3_URL" > >(tee -a $LOGFILE) 2> >(tee -a $LOGFILE >&2) || doLog "ERROR Unable to import S3Guard metadata."
```
Comment (keyki) [inline orchestrator-salt/src/main/resources/salt/salt/postgresql/disaster_recovery/scripts/restore_db.sh:132]: Is this line commented out on purpose?

Comment (kkalvagadda1) [reply orchestrator-salt/src/main/resources/salt/salt/postgresql/disaster_recovery/scripts/restore_db.sh:132]: Good catch. Nope. I just commented it as it was taking time in my testing. I will update the patch.

Comment (kkalvagadda1) [reply orchestrator-salt/src/main/resources/salt/salt/postgresql/disaster_recovery/scripts/restore_db.sh]: addressed it.

**Review by brycederriso — APPROVED** [review]
Changes look good to me.

**Pull request merged**